manage

```diff
@@ -54,9 +54,8 @@ EOF
 # -----------------------------------------------------------------------------------------------------------------
 # Default Settings:
 # -----------------------------------------------------------------------------------------------------------------
-DEFAULT_CONTAINERS="yoma-multitenant-wallets-db yoma-multitenant-web ngrok-yoma-multitenant yoma-multitenant-agent
-                    yoma-ga-wallets-db yoma-ga-web ngrok-yoma-ga yoma-ga-agent"
-                    # tails-server"
+DEFAULT_CONTAINERS="yoma-multitenant-wallets-db yoma-multitenant-agent
```

Comment:
notice how only the acapy agents are running. This is all that is actually required as the fastapi runs by itself when running tests - i.e. the tests start it.

I'd prefer to not have the fastapi running at all as we can just start fastapi via a fast api startup script and then also allow debugging of it. Debugging is challenging when running something on docker - or at least more challenging when running something natively.

Reply:
fastapi can be run with:

`uvicon main:app --reload --port 8082`

We get auto reload and we get a custom port and then if we run this in debug mode we can set break points...